### PR TITLE
fixes #24645 - remove unused brace dependencies

### PR DESCRIPTION
**Pull request by ohadlevy**
before:
```
vendor-d10491cbfddd4cacd626.js    4.14 MB       3  [emitted]  [big]  vendor
```
after:
```
vendor-0faa77d94a2f160c2733.js    3.15 MB       3  [emitted]  [big]  vendor
```

### CODE REVIEW

File: config/webpack.vendor.js [hunk 1]
```diff
@@ -43,12 +43,9 @@ module.exports = [
    * Brace related
    */
   'brace',
-  'brace/mode/javascript',
   'brace/mode/ruby',
-  'brace/mode/html_ruby',
```
Comment (tbrisker) [inline config/webpack.vendor.js]: Isn't this needed for erb?

File: config/webpack.vendor.js [hunk 2]
```diff
@@ -43,12 +43,9 @@ module.exports = [
...
   'brace/mode/json',
   'brace/mode/sh',
-  'brace/mode/xml',
```
Comment (tbrisker) [inline config/webpack.vendor.js]: xml may be useful, some templates use it (for example, AutoYaST SLES default)

**Review by tbrisker — COMMENTED** [review]
other than the js syntax, i'm not sure the others aren't used

File: config/webpack.vendor.js (hunk 1 above)
Comment (ohadlevy) [reply config/webpack.vendor.js]: thats what i thought initially when i added it, when testing it seems like its not.

File: config/webpack.vendor.js (hunk 2 above)
Comment (ohadlevy) [reply config/webpack.vendor.js]: that was the original reason I've added it, but i'm not sure its worth extra 300k for it..?

**Comment by ohadlevy** [comment]
@tbrisker added xml back, its 75K uncompressed and around 20k compressed.

**Comment by ohadlevy** [comment]
test failures seems not related?

**Review by tbrisker — CHANGES_REQUESTED** [review]
I'm afraid html_ruby is needed for erb, here is an example with erb highlighting selected.
Before this change:
screenshot from 2018-08-20 17-03-58

After the change:
screenshot from 2018-08-20 17-02-44

**Comment by ohadlevy** [comment]
Updated

**Review by tbrisker — APPROVED** [review]
Thanks @ohadlevy !

**Pull request merged**